.travis.yml

```diff
@@ -25,7 +25,9 @@ addons:
 
 before_install: yarn global add greenkeeper-lockfile@1
 before_script: greenkeeper-lockfile-update
-after_script: greenkeeper-lockfile-upload
+after_script:
+  - greenkeeper-lockfile-upload
+  - bash ./gh_pages.sh
```

Comment:
Use https://docs.travis-ci.com/user/deployment/script/ instead. Travis allows you to specify the branch on which it should run too.